src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -502,6 +503,7 @@ static String getResource(String key) {
         {"R_crClientUnrecoverable","The connection is broken and recovery is not possible. The connection is marked by the client driver as unrecoverable. No attempt was made to restore the connection."},
         {"R_crClientSSLStateNotRecoverable", "The server did not preserve SSL encryption during a recovery attempt, connection recovery is not possible."},
         {"R_crCommandCannotTimeOut", "Request failed to time out and SQLServerConnection does not exist"},
+        {"R_InvalidIPAddressPreference", "Invalid option for connection string option IPAddressPreference, the available options are: IPv4First, IPv6First and UsePlatformDefault"},
```

Comment:
add {0} on what the invalid option was, makes it easier to debug failures. Also, we don't usually list all the valid options here in the error string, in case we need to add/change anything

Reply:
Agreed, changes made.